src/EnrollChoicesController.js

```diff
@@ -202,8 +202,14 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
 
       this.state.set('pageType', pageType);
 
-      if (this.options.appState.get('skipLink') && pageType === Enums.HAS_REQUIRED_SOME_REQUIRED_ENROLLED) {
-        this.add(new Footer(this.toJSON()));
+      var showSkipLink = this.options.appState.get('skipLink')
+        && pageType === Enums.HAS_REQUIRED_SOME_REQUIRED_ENROLLED;
+      var showBackLink = this.settings.get('features.showCustomizableBackLinkInMFA');
```

Comment:
nit: lets start using const :)

Reply:
I agree we should, but I kind of like to be consistent as well, so I prefer to keep this file with the style it has today. For new files I will definitely do that